locustfile.py

```diff
@@ -164,6 +171,17 @@ def upload(self):
         if form:
             self.upload_addon(form)
 
+    @task(5)
```

Comment:
I think 5 times more might be a bit much since we mainly want to stress uploads (for now). Maybe twice as much?